cypress/components/tabs/tabs.cy.tsx

```diff
@@ -275,7 +278,7 @@ context("Testing Tabs component", () => {
 
         tabById(1)
           .children()
-          .children(child)
+          .children()
           .children()
           .should("have.attr", "data-component", "pill")
           .and("have.text", "12");
```

Comment:
At line 286-288 below (not something changed in this PR), you could change the variables in the array to be `validationType[0]`, `validationType[1]`, `validationType[2]` as you did in line 318.

Reply:
applied